files/usr/local/www/zeek_alerts.php

```diff
@@ -113,17 +113,23 @@
 });
 
 function updateSelect() {
-	var x = 'x='+$('#logfile').length;
-	jQuery.ajax({
-		type: "POST",
-		url: "select_box_file.php",
-		data: x,
-		success: function(html) {
-			if (html) {
-				$("#logfile").html(html);
-			}
-		},
-	});
+        var selected = $('#logfile').val();
+
+        var x = 'x='+$('#logfile').length;
+        jQuery.ajax({
+                type: "POST",
+                url: "select_box_file.php",
+                data: x,
+                success: function(html) {
+                        if (html) {
+                                $("#logfile").html(html);
+
+                                if (selected != 0 && selected != null && selected != undefined) {
```

Comment:
I suggest you to use this condition because it will cover cases where selected is null, 0, undefined, empty string, false or NaN.
`if (typeof selected !== 'undefined' && selected) {
}
`

Reply:
Good point, modified.